__tests__/help.ts

```diff
@@ -17,6 +19,11 @@ test('npx unimported --help', (done) => {
         --flow, -f    indicates if your code is annotated with flow types    [boolean]
         --update, -u  update the ignore-lists stored in .unimportedrc.json   [boolean]"
     `);
-    done();
-  });
+});
+
+test('error for wrong file name npx unimported --help', async () => {
+  const command = exec('node ./bin/1unimported.js --help');
```

Comment:
I've taken a closer look at what you're testing here and I don't think this test case valid.  All we're doing is validating `node` throws an error when the script file doesn't exist - nothing to do with `unimported`.

The changes in the other test to use promise are fine, so I think just remove this test all together and it should be good to go now.